src/components/Reports/Reports.js

```diff
@@ -111,7 +111,7 @@ const Reports = ({ dependencies: { datahubClient } }) => {
                     dateTo={state.dateTo}
                     dateFrom={state.dateFrom}
                     today={state.dailyView}
-                    emailFilter="with_email"
+                    emailFilter={'with_email'}
```

Comment:
What is this?

Reply:
It's for show the box with the data with email or without email

Reply:
Yes but it shouldn't  be part of this PR.

Reply:
I've talked this with @GusBaamonde and he did this change to be consistent with the other component, `without_email`